PKG-INFO

```diff
@@ -1,5 +1,5 @@
-Metadata-Version: 1.8.0
+Metadata-Version: 1.9
 Name: purity_fb
-Version: 1.8.0
-Summary: A python client for Purity//FB REST API.
+Version: 1.9
```

Comment:
can you make this "1.9.0"? it makes it easier if we find a bug in the client and need to update it